{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import myvariant"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_uri_list(context):\n",
    "    uri_path_dict = {}\n",
    "\n",
    "    for path, v in context.items():\n",
    "        for field_name, value in v[\"@context\"].items():\n",
    "            new_path = path.replace(\"/\", \".\") + \".\" + field_name\n",
    "            if value not in uri_path_dict:\n",
    "                uri_path_dict[value] = [new_path]\n",
    "            else:\n",
    "                uri_path_dict[value].append(new_path)\n",
    "    return uri_path_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def query_by_uri(uri, value, context):\n",
    "    mv = myvariant.MyVariantInfo()\n",
    "    context.pop('root')\n",
    "    query_string = ''\n",
    "    path_list = get_uri_list(context)[uri]\n",
    "    for _item in path_list:\n",
    "        query_string = query_string + ' ' + _item + ':' + value + ' OR'\n",
    "    query_string = query_string.strip(' OR')\n",
    "    return mv.query(query_string)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "context = json.loads(open(\"/Users/kevinxin/Documents/myvariant.info/json-ld/myvariant_context.json\").read())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'http://identifers.org/clinvar/': ['clinvar.rcv.accession'],\n",
       " 'http://identifiers.org/ccds/': ['cadd.gene.ccds_id'],\n",
       " 'http://identifiers.org/dbsnp/': ['root.rsid'],\n",
       " 'http://identifiers.org/efo/': ['clinvar.rcv.conditions.identifiers.efo'],\n",
       " 'http://identifiers.org/omim/': ['clinvar.omim',\n",
       "  'clinvar.rcv.conditions.identifiers.omim'],\n",
       " 'http://identifiers.org/orphanet/': ['clinvar.rcv.conditions.identifiers.orphanet'],\n",
       " 'http://identifiers.org/refseq/': ['evs.gene.accession', 'snpeff.feature_id'],\n",
       " 'http://localhost:8899/ensembl/': ['dbnsfp.ensembl.geneid',\n",
       "  'cadd.gene.gene_id',\n",
       "  'docm.ensembl_gene_id'],\n",
       " 'http://localhost:8899/gene/': ['dbsnp.gene.geneid'],\n",
       " 'http://localhost:8899/symbol/': ['evs.gene.symbol',\n",
       "  'clinvar.gene.symbol',\n",
       "  'root.genename',\n",
       "  'snpeff.ann.gene_name',\n",
       "  'dbsnp.gene.symbol',\n",
       "  'snpeff.lof.gene_name',\n",
       "  'docm.default_gene_name',\n",
       "  'emv.gene',\n",
       "  'snpeff.nmd.gene_name'],\n",
       " 'http://localhost:8899/uniprot/': ['dbnsfp.uniprot.acc', 'clinvar.uniprot'],\n",
       " 'http://schema.myvariant.info/datanode/ann': ['root.ann'],\n",
       " 'http://schema.myvariant.info/datanode/conditions': ['root.conditions'],\n",
       " 'http://schema.myvariant.info/datanode/ensembl': ['root.ensembl'],\n",
       " 'http://schema.myvariant.info/datanode/gene': ['root.gene'],\n",
       " 'http://schema.myvariant.info/datanode/identifiers': ['root.identifiers'],\n",
       " 'http://schema.myvariant.info/datanode/lof': ['root.lof'],\n",
       " 'http://schema.myvariant.info/datanode/nmd': ['root.nmd'],\n",
       " 'http://schema.myvariant.info/datanode/rcv': ['root.rcv'],\n",
       " 'http://schema.myvariant.info/datanode/uniprot': ['root.uniprot'],\n",
       " 'http://schema.myvariant.info/datasource/cadd': ['root.cadd'],\n",
       " 'http://schema.myvariant.info/datasource/clinvar': ['root.clinvar'],\n",
       " 'http://schema.myvariant.info/datasource/dbnsfp': ['root.dbnsfp'],\n",
       " 'http://schema.myvariant.info/datasource/dbsnp': ['root.dbsnp'],\n",
       " 'http://schema.myvariant.info/datasource/docm': ['root.docm'],\n",
       " 'http://schema.myvariant.info/datasource/emv': ['root.emv'],\n",
       " 'http://schema.myvariant.info/datasource/evs': ['root.evs'],\n",
       " 'http://schema.myvariant.info/datasource/gwassnps': ['root.gwassnps'],\n",
       " 'http://schema.myvariant.info/datasource/mutdb': ['root.mutdb'],\n",
       " 'http://schema.myvariant.info/datasource/snpeff': ['root.snpeff']}"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "get_uri_list(context)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'hits': [{'_id': 'chr12:g.56365348C>T',\n",
       "   '_score': 2.3620462,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'C',\n",
       "    'annotype': ['CodingTranscript', 'Intergenic', 'Intergenic', 'Transcript'],\n",
       "    'bstatistic': 238,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.165,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.244,\n",
       "     'txflnk': 0.047,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['missense', 'upstream', 'upstream', 'intron'],\n",
       "    'consequence': ['NON_SYNONYMOUS', 'UPSTREAM', 'UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [7, 1, 1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': 1.73, 'mgw': 0.24, 'prot': -3.09, 'roll': 0.45},\n",
       "    'encode': {'exp': 608.53,\n",
       "     'h3k27ac': 31.88,\n",
       "     'h3k4me1': 20.08,\n",
       "     'h3k4me3': 14.24,\n",
       "     'nucleo': 2.3},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'exon': '7/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.52,\n",
       "    'gene': [{'ccds_id': 'CCDS8898.1',\n",
       "      'cds': {'cdna_pos': 1076,\n",
       "       'cds_pos': 836,\n",
       "       'rel_cdna_pos': 0.47,\n",
       "       'rel_cds_pos': 0.93},\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2',\n",
       "      'prot': {'domain': 'ndomain', 'protpos': 279, 'rel_prot_pos': 0.94}},\n",
       "     {'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'feature_id': 'ENST00000549233',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'}],\n",
       "    'gerp': {'n': 5.37, 'rs': 466.9, 'rs_pval': 8.6458e-62, 's': 5.37},\n",
       "    'grantham': 64,\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 62,\n",
       "    'min_dist_tss': 444,\n",
       "    'mutindex': -49,\n",
       "    'naa': 'V',\n",
       "    'oaa': 'A',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.993, 'vertebrate': 1.0},\n",
       "    'phred': 23.2,\n",
       "    'phylop': {'mammalian': 2.685, 'primate': 0.587, 'vertebrate': 2.761},\n",
       "    'polyphen': {'cat': 'benign', 'val': 0.002},\n",
       "    'pos': 56365348,\n",
       "    'rawscore': 3.610169,\n",
       "    'ref': 'C',\n",
       "    'segway': 'GM1',\n",
       "    'sift': {'cat': 'tolerated', 'val': 0.17},\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'V',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 2,\n",
       "     'pos': ['279', '327', '219', '245'],\n",
       "     'ref': 'A',\n",
       "     'refcodon': 'GCA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'C',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.46804,\n",
       "     'score': 0.9884751589295847},\n",
       "    'eigen': {'phred': 2.487573,\n",
       "     'raw': -0.00665388086091324,\n",
       "     'raw_rankscore': 0.4162},\n",
       "    'eigen-pc': {'raw': 0.196375111796387, 'raw_rankscore': 0.49965},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'esp6500': {'aa_ac': 0,\n",
       "     'aa_af': 0.0,\n",
       "     'ea_ac': 1,\n",
       "     'ea_af': 0.00011627906976744187},\n",
       "    'fathmm': {'pred': ['T', 'T', 'T', 'T'],\n",
       "     'rankscore': 0.44858,\n",
       "     'score': [0.91, 0.91, 0.91, 0.91]},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.63311,\n",
       "     'coding_score': 0.9513},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999909310465},\n",
       "    'gerp++': {'nr': 5.37, 'rs': 5.37, 'rs_rankscore': 0.7686},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651615, 'start': 54651615},\n",
       "    'hg19': {'end': 56365348, 'start': 56365348},\n",
       "    'hg38': {'end': 55971564, 'start': 55971564},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.55863,\n",
       "     'omega': 0.120484,\n",
       "     'pred': 'D',\n",
       "     'score': 3.5e-05},\n",
       "    'metalr': {'pred': 'T', 'rankscore': 0.35805, 'score': 0.0946},\n",
       "    'metasvm': {'pred': 'T', 'rankscore': 0.27966, 'score': -1.0066},\n",
       "    'mutationassessor': {'pred': 'L', 'rankscore': 0.3892, 'score': 1.53},\n",
       "    'mutationtaster': {'AAE': ['A279V', 'A327V', 'A219V', 'A245V'],\n",
       "     'converted_rankscore': 0.45119,\n",
       "     'model': ['simple_aae', 'simple_aae', 'simple_aae', 'simple_aae'],\n",
       "     'pred': ['D', 'D', 'D', 'N'],\n",
       "     'score': [0.998571, 0.985721, 0.795136, 0.610929]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.998, 'mammalian_rankscore': 0.69722}},\n",
       "    'phylo': {'p100way': {'vertebrate': 4.847,\n",
       "      'vertebrate_rankscore': 0.62331},\n",
       "     'p20way': {'mammalian': 0.789, 'mammalian_rankscore': 0.31954}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'pred': 'B',\n",
       "      'rankscore': 0.11171,\n",
       "      'score': [0.001, 0.004, 0.001]},\n",
       "     'hvar': {'pred': 'B',\n",
       "      'rankscore': 0.10362,\n",
       "      'score': [0.005, 0.005, 0.002]}},\n",
       "    'provean': {'pred': 'N',\n",
       "     'rankscore': 0.3735,\n",
       "     'score': [-1.54, -1.48, -1.29, -1.29]},\n",
       "    'ref': 'C',\n",
       "    'reliability_index': 10,\n",
       "    'rsid': 'rs142099990',\n",
       "    'sift': {'converted_rankscore': 0.20501,\n",
       "     'pred': 'T',\n",
       "     'score': [0.275, 0.198, 0.278, 0.277]},\n",
       "    'siphy_29way': {'logodds': 18.2571,\n",
       "     'logodds_rankscore': 0.89915,\n",
       "     'pi': {'a': 0.0, 'c': 1.0, 'g': 0.0, 't': 0.0}},\n",
       "    'uniprot': [{'acc': 'E7ESI2', 'pos': '219'},\n",
       "     {'acc': 'P24941-2', 'pos': '245'},\n",
       "     {'acc': 'P24941', 'pos': '279'}],\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F',\n",
       "     'rankscore': ['0.345', '0.35', '0.35', '0.345', '0.345', '0.292'],\n",
       "     'score': [0.345, 0.35, 0.35, 0.345, 0.345, 0.292],\n",
       "     'transcriptid': ['ENST00000440311',\n",
       "      'NM_001798.3',\n",
       "      'ENST00000266970',\n",
       "      'ENST00000354056',\n",
       "      'NM_052827.2',\n",
       "      'ENST00000553376'],\n",
       "     'transcriptvar': ['A219V', 'A279V', 'A279V', 'A245V', 'A245V', 'A327V']}},\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'C'}, {'allele': 'T'}],\n",
       "    'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 134,\n",
       "    'flags': ['ASP', 'NSM', 'REF', 'S3D'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56365348, 'start': 56365348},\n",
       "    'ref': 'C',\n",
       "    'rsid': 'rs142099990',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'C': 4406, 'T': 0},\n",
       "     'all': {'C': 13005, 'T': 1},\n",
       "     'european_american': {'C': 8599, 'T': 1}},\n",
       "    'alt': 'T',\n",
       "    'avg_sample_read': 137,\n",
       "    'chimp_allele': 'C',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 5.4, 'phast_cons': 1.0},\n",
       "    'dbsnp_version': 134,\n",
       "    'estimated_age_kyrs': {'ea': '1.2+/-3.3'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'missense',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'CC': 2203, 'TC': 0, 'TT': 0},\n",
       "     'all_genotype': {'CC': 6502, 'TC': 1, 'TT': 0},\n",
       "     'european_american': {'CC': 4299, 'TC': 1, 'TT': 0}},\n",
       "    'grantham_score': 64,\n",
       "    'hg19': {'end': 56365348, 'start': 56365348},\n",
       "    'hg38': {'end': 55971564, 'start': 55971564},\n",
       "    'hgvs': [{'coding': 'c.734C>T', 'protein': 'p.(A245V)'},\n",
       "     {'coding': 'c.836C>T', 'protein': 'p.(A279V)'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0,\n",
       "     'all': 0.0077,\n",
       "     'european_american': 0.0116},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'b', 'score': 'e'},\n",
       "    'ref': 'C',\n",
       "    'ref_base_ncbi': 'C',\n",
       "    'rsid': 'rs142099990'},\n",
       "   'hg19': {'end': 56365348, 'start': 56365348},\n",
       "   'mutdb': {'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'hg19': {'end': 56365348, 'start': 56365348},\n",
       "    'mutpred_score': -1.0,\n",
       "    'ref': 'C',\n",
       "    'rsid': 'rs142099990',\n",
       "    'strand': 'p'},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '1076'},\n",
       "      'cds': {'length': '897', 'position': '836'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.836C>T',\n",
       "      'hgvs_p': 'p.Ala279Val',\n",
       "      'protein': {'length': '298', 'position': '279'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '7',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '896'},\n",
       "      'cds': {'length': '717', 'position': '656'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.656C>T',\n",
       "      'hgvs_p': 'p.Ala219Val',\n",
       "      'protein': {'length': '238', 'position': '219'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '5',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '974'},\n",
       "      'cds': {'length': '795', 'position': '734'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.734C>T',\n",
       "      'hgvs_p': 'p.Ala245Val',\n",
       "      'protein': {'length': '264', 'position': '245'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '6',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2349',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4852',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2447',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2447',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'T', 'position': '56365348', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56363219C>G',\n",
       "   '_score': 2.3573928,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'C',\n",
       "    'annotype': ['Intergenic', 'Transcript'],\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.063,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.268,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.583,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['upstream', 'intron'],\n",
       "    'consequence': ['UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': 3.01, 'mgw': 0.04, 'prot': 0.28, 'roll': -1.2},\n",
       "    'encode': {'exp': 82.79,\n",
       "     'h3k27ac': 4.92,\n",
       "     'h3k4me1': 7.28,\n",
       "     'h3k4me3': 4.08,\n",
       "     'nucleo': 1.5},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'fitcons': 0.117096,\n",
       "    'gc': 0.53,\n",
       "    'gene': [{'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'}],\n",
       "    'gerp': {'n': 5.44, 's': 0.344},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 449,\n",
       "    'min_dist_tss': 1973,\n",
       "    'motif': {'dist': -0.1, 'toverlap': 2},\n",
       "    'mutindex': -20,\n",
       "    'phast_cons': {'mammalian': 0.0, 'primate': 0.004, 'vertebrate': 0.0},\n",
       "    'phred': 2.498,\n",
       "    'phylop': {'mammalian': -0.044, 'primate': -0.926, 'vertebrate': 0.072},\n",
       "    'pos': 56363219,\n",
       "    'rawscore': -0.009279,\n",
       "    'ref': 'C',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'C'}, {'allele': 'G'}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 138,\n",
       "    'flags': ['ASP', 'INT'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56363219, 'start': 56363219},\n",
       "    'ref': 'C',\n",
       "    'rsid': 'rs374494562',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'tv',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'C': 4405, 'G': 1},\n",
       "     'all': {'C': 13005, 'G': 1},\n",
       "     'european_american': {'C': 8600, 'G': 0}},\n",
       "    'alt': 'G',\n",
       "    'avg_sample_read': 45,\n",
       "    'chimp_allele': 'C',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 0.3, 'phast_cons': 0.0},\n",
       "    'dbsnp_version': 138,\n",
       "    'estimated_age_kyrs': {'aa': '2.1+/-5.4'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'CC': 2202, 'GC': 1, 'GG': 0},\n",
       "     'all_genotype': {'CC': 6502, 'GC': 1, 'GG': 0},\n",
       "     'european_american': {'CC': 4300, 'GC': 0, 'GG': 0}},\n",
       "    'hg19': {'end': 56363219, 'start': 56363219},\n",
       "    'hg38': {'end': 55969435, 'start': 55969435},\n",
       "    'hgvs': [{'coding': 'c.486+487C>G'}, {'coding': 'c.487-40C>G'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0227,\n",
       "     'all': 0.0077,\n",
       "     'european_american': 0.0},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'C',\n",
       "    'ref_base_ncbi': 'C',\n",
       "    'rsid': 'rs374494562'},\n",
       "   'exac': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 1,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 0,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 8.238e-06,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 121396,\n",
       "     'an_adj': 113426,\n",
       "     'an_afr': 10172,\n",
       "     'an_amr': 11274,\n",
       "     'an_eas': 8514,\n",
       "     'an_female': 51582,\n",
       "     'an_fin': 6500,\n",
       "     'an_male': 61844,\n",
       "     'an_nfe': 64282,\n",
       "     'an_oth': 826,\n",
       "     'an_sas': 11858},\n",
       "    'baseqranksum': -4.461,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -1.629,\n",
       "    'culprit': 'MQ',\n",
       "    'dp': 1534188,\n",
       "    'fs': 5.097,\n",
       "    'het': {'ac_het': 1,\n",
       "     'het_afr': 1,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 0,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 0,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0015,\n",
       "    'mq': {'mq': 60.0, 'mq0': 0, 'mqranksum': 0.015},\n",
       "    'ncc': 31,\n",
       "    'pos': 56363219,\n",
       "    'qd': 16.61,\n",
       "    'qual': 1312.52,\n",
       "    'readposranksum': 1.62,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 1.06},\n",
       "   'exac_nontcga': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 1,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 0,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 9.417e-06,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 106194,\n",
       "     'an_adj': 98434,\n",
       "     'an_afr': 8838,\n",
       "     'an_amr': 10918,\n",
       "     'an_eas': 7736,\n",
       "     'an_female': 43410,\n",
       "     'an_fin': 6500,\n",
       "     'an_male': 55024,\n",
       "     'an_nfe': 52064,\n",
       "     'an_oth': 622,\n",
       "     'an_sas': 11756},\n",
       "    'baseqranksum': -4.461,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -1.629,\n",
       "    'culprit': 'MQ',\n",
       "    'fs': 5.097,\n",
       "    'het': {'het_afr': 1,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 0,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0015,\n",
       "    'mq': {'mq': 60.0, 'mq0': 0, 'mqranksum': 0.015},\n",
       "    'ncc': 31,\n",
       "    'pos': 56363219,\n",
       "    'qd': 16.61,\n",
       "    'readposranksum': 1.62,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 1.06},\n",
       "   'hg19': {'end': 56363219, 'start': 56363219},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '3373',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4478',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3373',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2723',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4576',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4576',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.487-40C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+487C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+487C>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'G', 'position': '56363219', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56363222A>G',\n",
       "   '_score': 2.357366,\n",
       "   'cadd': {'1000g': {'af': 0.003, 'afr': 0.002, 'amr': 0.003, 'eur': 0.01},\n",
       "    '_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'A',\n",
       "    'annotype': ['Intergenic', 'Transcript'],\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.063,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.268,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.583,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['upstream', 'intron'],\n",
       "    'consequence': ['UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': 0.62, 'mgw': 0.3, 'prot': 2.07, 'roll': 1.91},\n",
       "    'encode': {'exp': 91.49,\n",
       "     'h3k27ac': 4.92,\n",
       "     'h3k4me1': 7.28,\n",
       "     'h3k4me3': 4.08,\n",
       "     'nucleo': 1.6},\n",
       "    'esp': {'af': 0.006, 'afr': 0.002, 'eur': 0.008},\n",
       "    'fitcons': 0.117096,\n",
       "    'gc': 0.54,\n",
       "    'gene': [{'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'}],\n",
       "    'gerp': {'n': 4.92, 's': -4.81},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 446,\n",
       "    'min_dist_tss': 1976,\n",
       "    'mutindex': -28,\n",
       "    'phast_cons': {'mammalian': 0.0, 'primate': 0.042, 'vertebrate': 0.0},\n",
       "    'phred': 6.489,\n",
       "    'phylop': {'mammalian': -0.97, 'primate': 0.525, 'vertebrate': -0.711},\n",
       "    'pos': 56363222,\n",
       "    'rawscore': 0.385723,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'A', 'freq': 0.9954},\n",
       "     {'allele': 'G', 'freq': 0.004593}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 130,\n",
       "    'flags': ['ASP', 'HD', 'INT', 'KGPhase1', 'KGPhase3', 'SLO'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'gmaf': 0.004593,\n",
       "    'hg19': {'end': 56363222, 'start': 56363222},\n",
       "    'ref': 'A',\n",
       "    'rsid': 'rs73123218',\n",
       "    'validated': True,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'A': 4398, 'G': 8},\n",
       "     'all': {'A': 12927, 'G': 79},\n",
       "     'european_american': {'A': 8529, 'G': 71}},\n",
       "    'alt': 'G',\n",
       "    'avg_sample_read': 46,\n",
       "    'chimp_allele': 'A',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': -4.8, 'phast_cons': 0.0},\n",
       "    'dbsnp_version': 130,\n",
       "    'estimated_age_kyrs': {'aa': '14.3+/-33.2', 'ea': '42.6+/-100.3'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'AA': 2195, 'GA': 8, 'GG': 0},\n",
       "     'all_genotype': {'AA': 6425, 'GA': 77, 'GG': 1},\n",
       "     'european_american': {'AA': 4230, 'GA': 69, 'GG': 1}},\n",
       "    'hg19': {'end': 56363222, 'start': 56363222},\n",
       "    'hg38': {'end': 55969438, 'start': 55969438},\n",
       "    'hgvs': [{'coding': 'c.486+490A>G'}, {'coding': 'c.487-37A>G'}],\n",
       "    'ma_fin_percent': {'african_american': 0.1816,\n",
       "     'all': 0.6074,\n",
       "     'european_american': 0.8256},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'A',\n",
       "    'ref_base_ncbi': 'A',\n",
       "    'rsid': 'rs73123218'},\n",
       "   'exac': {'ac': {'ac': 737,\n",
       "     'ac_adj': 707,\n",
       "     'ac_afr': 14,\n",
       "     'ac_amr': 20,\n",
       "     'ac_eas': 2,\n",
       "     'ac_female': 287,\n",
       "     'ac_fin': 13,\n",
       "     'ac_het': 689,\n",
       "     'ac_hom': 9,\n",
       "     'ac_male': 420,\n",
       "     'ac_nfe': 486,\n",
       "     'ac_oth': 3,\n",
       "     'ac_sas': 169},\n",
       "    'af': 0.006071,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 121394,\n",
       "     'an_adj': 113724,\n",
       "     'an_afr': 10188,\n",
       "     'an_amr': 11282,\n",
       "     'an_eas': 8532,\n",
       "     'an_female': 51708,\n",
       "     'an_fin': 6504,\n",
       "     'an_male': 62016,\n",
       "     'an_nfe': 64384,\n",
       "     'an_oth': 834,\n",
       "     'an_sas': 12000},\n",
       "    'baseqranksum': 3.44,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.046,\n",
       "    'culprit': 'MQ',\n",
       "    'dp': 1554195,\n",
       "    'fs': 0.0,\n",
       "    'het': {'ac_het': 689,\n",
       "     'het_afr': 14,\n",
       "     'het_amr': 20,\n",
       "     'het_eas': 2,\n",
       "     'het_fin': 13,\n",
       "     'het_nfe': 478,\n",
       "     'het_oth': 3,\n",
       "     'het_sas': 159},\n",
       "    'hom': {'ac_hom': 9,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 4,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 5},\n",
       "    'inbreedingcoeff': 0.0157,\n",
       "    'mq': {'mq': 60.0, 'mq0': 0, 'mqranksum': 0.248},\n",
       "    'ncc': 29,\n",
       "    'pos': 56363222,\n",
       "    'qd': 16.07,\n",
       "    'qual': 575413.93,\n",
       "    'readposranksum': 0.453,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 6.12},\n",
       "   'exac_nontcga': {'ac': {'ac': 628,\n",
       "     'ac_adj': 598,\n",
       "     'ac_afr': 12,\n",
       "     'ac_amr': 20,\n",
       "     'ac_eas': 1,\n",
       "     'ac_female': 227,\n",
       "     'ac_fin': 13,\n",
       "     'ac_het': 582,\n",
       "     'ac_hom': 8,\n",
       "     'ac_male': 371,\n",
       "     'ac_nfe': 385,\n",
       "     'ac_oth': 2,\n",
       "     'ac_sas': 165},\n",
       "    'af': 0.005914,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 106192,\n",
       "     'an_adj': 98702,\n",
       "     'an_afr': 8852,\n",
       "     'an_amr': 10926,\n",
       "     'an_eas': 7754,\n",
       "     'an_female': 43526,\n",
       "     'an_fin': 6504,\n",
       "     'an_male': 55176,\n",
       "     'an_nfe': 52138,\n",
       "     'an_oth': 628,\n",
       "     'an_sas': 11900},\n",
       "    'baseqranksum': 3.44,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.046,\n",
       "    'culprit': 'MQ',\n",
       "    'fs': 0.0,\n",
       "    'het': {'het_afr': 12,\n",
       "     'het_amr': 20,\n",
       "     'het_eas': 1,\n",
       "     'het_fin': 13,\n",
       "     'het_nfe': 379,\n",
       "     'het_oth': 2,\n",
       "     'het_sas': 155},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 3,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 5},\n",
       "    'inbreedingcoeff': 0.0157,\n",
       "    'mq': {'mq': 60.0, 'mq0': 0, 'mqranksum': 0.248},\n",
       "    'ncc': 29,\n",
       "    'pos': 56363222,\n",
       "    'qd': 16.07,\n",
       "    'readposranksum': 0.453,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 6.12},\n",
       "   'geno2mp': {'hpo_count': 10},\n",
       "   'hg19': {'end': 56363222, 'start': 56363222},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '3376',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4475',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3376',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2726',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4573',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4573',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.487-37A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+490A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+490A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'G', 'position': '56363222', 'ref': 'A'},\n",
       "   'wellderly': {'alleles': [{'allele': 'A', 'freq': 0.995},\n",
       "     {'allele': 'G', 'freq': 0.005}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'gene': 'CDK2',\n",
       "    'genotypes': [{'count': 2, 'freq': 0.01, 'genotype': 'A/G'},\n",
       "     {'count': 198, 'freq': 0.99, 'genotype': 'A/A'}],\n",
       "    'hg19': {'end': 56363222, 'start': 56363222},\n",
       "    'pos': 56363222,\n",
       "    'ref': 'A',\n",
       "    'vartype': 'snp'}},\n",
       "  {'_id': 'chr12:g.56363230del',\n",
       "   '_score': 2.3528328,\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'AC'}, {'allele': 'A'}],\n",
       "    'alt': 'A',\n",
       "    'chrom': '12',\n",
       "    'class': 'DIV',\n",
       "    'dbsnp_build': 144,\n",
       "    'flags': ['ASP', 'INT'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56363231, 'start': 56363230},\n",
       "    'ref': 'AC',\n",
       "    'rsid': 'rs777935868',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'del',\n",
       "    'vartype': 'indel'},\n",
       "   'evs': {'allele_count': {'african_american': {'A1': 5, 'R': 4259},\n",
       "     'all': {'A1': 23, 'R': 12495},\n",
       "     'european_american': {'A1': 18, 'R': 8236}},\n",
       "    'alt': 'A',\n",
       "    'avg_sample_read': 51,\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': -4.5, 'phast_cons': 0.0},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'A1A1': 2, 'A1R': 1, 'RR': 2129},\n",
       "     'all_genotype': {'A1A1': 10, 'A1R': 3, 'RR': 6246},\n",
       "     'european_american': {'A1A1': 8, 'A1R': 2, 'RR': 4117}},\n",
       "    'hg19': {'end': 56363230, 'start': 56363230},\n",
       "    'hg38': {'end': 55969446, 'start': 55969446},\n",
       "    'hgvs': [{'coding': 'c.486+498del1'}, {'coding': 'c.487-29del1'}],\n",
       "    'ma_fin_percent': {'african_american': 0.1173,\n",
       "     'all': 0.1837,\n",
       "     'european_american': 0.2181},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'AC',\n",
       "    'ref_base_ncbi': 'AC'},\n",
       "   'exac': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 1,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 0,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 1,\n",
       "     'ac_nfe': 0,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 8.237e-06,\n",
       "    'alleles': 'A',\n",
       "    'alt': 'A',\n",
       "    'an': {'an': 121404,\n",
       "     'an_adj': 113892,\n",
       "     'an_afr': 10172,\n",
       "     'an_amr': 11306,\n",
       "     'an_eas': 8562,\n",
       "     'an_female': 51712,\n",
       "     'an_fin': 6328,\n",
       "     'an_male': 62180,\n",
       "     'an_nfe': 64436,\n",
       "     'an_oth': 834,\n",
       "     'an_sas': 12254},\n",
       "    'baseqranksum': -3.752,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.412,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 281.975,\n",
       "    'het': {'het_afr': 1,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 0,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0025,\n",
       "    'mq': {'mq': 59.58, 'mq0': 0, 'mqranksum': -0.314},\n",
       "    'ncc': 16,\n",
       "    'pos': 56363229,\n",
       "    'qd': 13.64,\n",
       "    'readposranksum': 0.217,\n",
       "    'ref': 'AC',\n",
       "    'type': 'del',\n",
       "    'vqslod': -68.82},\n",
       "   'exac_nontcga': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 1,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 0,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 1,\n",
       "     'ac_nfe': 0,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 9.416e-06,\n",
       "    'alleles': 'A',\n",
       "    'alt': 'A',\n",
       "    'an': {'an': 106202,\n",
       "     'an_adj': 98916,\n",
       "     'an_afr': 8856,\n",
       "     'an_amr': 10950,\n",
       "     'an_eas': 7786,\n",
       "     'an_female': 43578,\n",
       "     'an_fin': 6328,\n",
       "     'an_male': 55338,\n",
       "     'an_nfe': 52216,\n",
       "     'an_oth': 628,\n",
       "     'an_sas': 12152},\n",
       "    'baseqranksum': -3.752,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.412,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 281.975,\n",
       "    'het': {'het_afr': 1,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 0,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0025,\n",
       "    'mq': {'mq': 59.58, 'mq0': 0, 'mqranksum': -0.314},\n",
       "    'ncc': 16,\n",
       "    'pos': 56363229,\n",
       "    'qd': 13.64,\n",
       "    'readposranksum': 0.217,\n",
       "    'ref': 'AC',\n",
       "    'type': 'del',\n",
       "    'vqslod': -68.82},\n",
       "   'hg19': {'end': 56363231, 'start': 56363230},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '3384',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52delG',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4467',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3384',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52delG',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2734',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87delG',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4565',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4565',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.487-29delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+498delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+498delC',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'A', 'position': '56363229', 'ref': 'AC'}},\n",
       "  {'_id': 'chr12:g.56365344A>G',\n",
       "   '_score': 2.3522885,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'A',\n",
       "    'annotype': ['CodingTranscript', 'Intergenic', 'Intergenic', 'Transcript'],\n",
       "    'bstatistic': 238,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.165,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.244,\n",
       "     'txflnk': 0.047,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['missense', 'upstream', 'upstream', 'intron'],\n",
       "    'consequence': ['NON_SYNONYMOUS', 'UPSTREAM', 'UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [7, 1, 1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': 0.03, 'mgw': 0.4, 'prot': 3.05, 'roll': 3.04},\n",
       "    'encode': {'exp': 561.33,\n",
       "     'h3k27ac': 31.88,\n",
       "     'h3k4me1': 20.08,\n",
       "     'h3k4me3': 14.24,\n",
       "     'nucleo': 2.3},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'exon': '7/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.52,\n",
       "    'gene': [{'ccds_id': 'CCDS8898.1',\n",
       "      'cds': {'cdna_pos': 1072,\n",
       "       'cds_pos': 832,\n",
       "       'rel_cdna_pos': 0.47,\n",
       "       'rel_cds_pos': 0.93},\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2',\n",
       "      'prot': {'domain': 'ndomain', 'protpos': 278, 'rel_prot_pos': 0.93}},\n",
       "     {'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'feature_id': 'ENST00000549233',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'}],\n",
       "    'gerp': {'n': 5.37, 'rs': 466.9, 'rs_pval': 8.6458e-62, 's': 5.37},\n",
       "    'grantham': 56,\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 66,\n",
       "    'min_dist_tss': 440,\n",
       "    'mutindex': -55,\n",
       "    'naa': 'E',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 0.999, 'primate': 0.987, 'vertebrate': 1.0},\n",
       "    'phred': 23.1,\n",
       "    'phylop': {'mammalian': 2.168, 'primate': 0.478, 'vertebrate': 4.816},\n",
       "    'polyphen': {'cat': 'benign', 'val': 0.147},\n",
       "    'pos': 56365344,\n",
       "    'rawscore': 3.509328,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GM1',\n",
       "    'sift': {'cat': 'tolerated', 'val': 0.07},\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'E',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['278', '326', '218', '244'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAG'},\n",
       "    'alt': 'G',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.81692,\n",
       "     'score': 0.9971821698746259},\n",
       "    'eigen': {'phred': 3.467034,\n",
       "     'raw': 0.236159841048146,\n",
       "     'raw_rankscore': 0.53011},\n",
       "    'eigen-pc': {'raw': 0.406376111871128, 'raw_rankscore': 0.62128},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'esp6500': {'aa_ac': 0,\n",
       "     'aa_af': 0.0,\n",
       "     'ea_ac': 1,\n",
       "     'ea_af': 0.00011627906976744187},\n",
       "    'exac': {'ac': 6,\n",
       "     'adj_ac': 6,\n",
       "     'adj_af': 4.943e-05,\n",
       "     'af': 4.942e-05,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 6,\n",
       "     'nfe_af': 8.992e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'exac_nonpsych': {'ac': 3,\n",
       "     'adj_ac': 3,\n",
       "     'adj_af': 3.306e-05,\n",
       "     'af': 3.306e-05,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 3,\n",
       "     'nfe_af': 7.121e-05},\n",
       "    'exac_nontcga': {'ac': 6,\n",
       "     'adj_ac': 6,\n",
       "     'adj_af': 5.65e-05,\n",
       "     'af': 5.649e-05,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 6,\n",
       "     'nfe_af': 0.0001104,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'fathmm': {'pred': ['T', 'T', 'T', 'T'],\n",
       "     'rankscore': 0.64463,\n",
       "     'score': [-0.11, -0.11, -0.11, -0.11]},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.95296,\n",
       "     'coding_score': 0.99384},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999998333},\n",
       "    'gerp++': {'nr': 5.37, 'rs': 5.37, 'rs_rankscore': 0.7686},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651611, 'start': 54651611},\n",
       "    'hg19': {'end': 56365344, 'start': 56365344},\n",
       "    'hg38': {'end': 55971560, 'start': 55971560},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'metalr': {'pred': 'T', 'rankscore': 0.54911, 'score': 0.1949},\n",
       "    'metasvm': {'pred': 'T', 'rankscore': 0.68503, 'score': -0.5047},\n",
       "    'mutationassessor': {'pred': 'N', 'rankscore': 0.01621, 'score': -0.795},\n",
       "    'mutationtaster': {'AAE': ['K278E', 'K326E', 'K218E', 'K244E'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['simple_aae', 'simple_aae', 'simple_aae', 'simple_aae'],\n",
       "     'pred': ['D', 'D', 'D', 'D'],\n",
       "     'score': [1, 1, 1, 0.999999]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.993, 'mammalian_rankscore': 0.57359}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.309,\n",
       "      'vertebrate_rankscore': 0.95275},\n",
       "     'p20way': {'mammalian': 1.005, 'mammalian_rankscore': 0.62273}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'pred': ['B', 'D', 'B'],\n",
       "      'rankscore': 0.68941,\n",
       "      'score': [0.364, 0.997, 0.421]},\n",
       "     'hvar': {'pred': ['B', 'D', 'B'],\n",
       "      'rankscore': 0.65931,\n",
       "      'score': [0.168, 0.941, 0.135]}},\n",
       "    'provean': {'pred': 'D',\n",
       "     'rankscore': 0.62478,\n",
       "     'score': [-2.84, -2.94, -3.01, -2.87]},\n",
       "    'ref': 'A',\n",
       "    'reliability_index': 10,\n",
       "    'rsid': 'rs148619120',\n",
       "    'sift': {'converted_rankscore': 0.78421,\n",
       "     'pred': 'D',\n",
       "     'score': [0.002, 0.001, 0.002, 0.002]},\n",
       "    'siphy_29way': {'logodds': 14.658,\n",
       "     'logodds_rankscore': 0.68315,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'uniprot': [{'acc': 'E7ESI2', 'pos': '218'},\n",
       "     {'acc': 'P24941-2', 'pos': '244'},\n",
       "     {'acc': 'P24941', 'pos': '278'}],\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F',\n",
       "     'rankscore': ['0.832', '0.832', '0.832', '0.832', '0.832', '0.857'],\n",
       "     'score': [0.832, 0.832, 0.832, 0.832, 0.832, 0.857],\n",
       "     'transcriptid': ['ENST00000440311',\n",
       "      'NM_001798.3',\n",
       "      'ENST00000266970',\n",
       "      'ENST00000354056',\n",
       "      'NM_052827.2',\n",
       "      'ENST00000553376'],\n",
       "     'transcriptvar': ['K218E', 'K278E', 'K278E', 'K244E', 'K244E', 'K326E']}},\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'A'}, {'allele': 'G'}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 134,\n",
       "    'flags': ['ASP', 'NSM', 'REF', 'S3D'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56365344, 'start': 56365344},\n",
       "    'ref': 'A',\n",
       "    'rsid': 'rs148619120',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'A': 4406, 'G': 0},\n",
       "     'all': {'A': 13005, 'G': 1},\n",
       "     'european_american': {'A': 8599, 'G': 1}},\n",
       "    'alt': 'G',\n",
       "    'avg_sample_read': 137,\n",
       "    'chimp_allele': 'A',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 5.4, 'phast_cons': 1.0},\n",
       "    'dbsnp_version': 134,\n",
       "    'estimated_age_kyrs': {'ea': '1.2+/-3.3'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'missense',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'AA': 2203, 'GA': 0, 'GG': 0},\n",
       "     'all_genotype': {'AA': 6502, 'GA': 1, 'GG': 0},\n",
       "     'european_american': {'AA': 4299, 'GA': 1, 'GG': 0}},\n",
       "    'grantham_score': 56,\n",
       "    'hg19': {'end': 56365344, 'start': 56365344},\n",
       "    'hg38': {'end': 55971560, 'start': 55971560},\n",
       "    'hgvs': [{'coding': 'c.730A>G', 'protein': 'p.(K244E)'},\n",
       "     {'coding': 'c.832A>G', 'protein': 'p.(K278E)'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0,\n",
       "     'all': 0.0077,\n",
       "     'european_american': 0.0116},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'p', 'score': 'r'},\n",
       "    'ref': 'A',\n",
       "    'ref_base_ncbi': 'A',\n",
       "    'rsid': 'rs148619120'},\n",
       "   'exac': {'ac': {'ac': 6,\n",
       "     'ac_adj': 6,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 3,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 6,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 3,\n",
       "     'ac_nfe': 6,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 4.942e-05,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121396,\n",
       "     'an_afr': 10406,\n",
       "     'an_amr': 11578,\n",
       "     'an_eas': 8654,\n",
       "     'an_female': 54110,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 67286,\n",
       "     'an_nfe': 66724,\n",
       "     'an_oth': 908,\n",
       "     'an_sas': 16512},\n",
       "    'baseqranksum': -0.931,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.001,\n",
       "    'culprit': 'InbreedingCoeff',\n",
       "    'dp': 2330395,\n",
       "    'fs': 0.538,\n",
       "    'het': {'ac_het': 6,\n",
       "     'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 6,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 0,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0001,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': 1.28},\n",
       "    'ncc': 0,\n",
       "    'pos': 56365344,\n",
       "    'qd': 10.73,\n",
       "    'qual': 13957.12,\n",
       "    'readposranksum': 0.069,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 1.22},\n",
       "   'exac_nontcga': {'ac': {'ac': 6,\n",
       "     'ac_adj': 6,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 3,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 6,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 3,\n",
       "     'ac_nfe': 6,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 5.649e-05,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 106194,\n",
       "     'an_afr': 9066,\n",
       "     'an_amr': 11216,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45860,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 60334,\n",
       "     'an_nfe': 54330,\n",
       "     'an_oth': 694,\n",
       "     'an_sas': 16408},\n",
       "    'baseqranksum': -0.931,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.001,\n",
       "    'culprit': 'InbreedingCoeff',\n",
       "    'fs': 0.538,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 6,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0001,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': 1.28},\n",
       "    'ncc': 0,\n",
       "    'pos': 56365344,\n",
       "    'qd': 10.73,\n",
       "    'readposranksum': 0.069,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 1.22},\n",
       "   'hg19': {'end': 56365344, 'start': 56365344},\n",
       "   'mutdb': {'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'hg19': {'end': 56365344, 'start': 56365344},\n",
       "    'mutpred_score': -1.0,\n",
       "    'ref': 'A',\n",
       "    'rsid': 'rs148619120',\n",
       "    'strand': 'p'},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '1072'},\n",
       "      'cds': {'length': '897', 'position': '832'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.832A>G',\n",
       "      'hgvs_p': 'p.Lys278Glu',\n",
       "      'protein': {'length': '298', 'position': '278'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '7',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '892'},\n",
       "      'cds': {'length': '717', 'position': '652'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.652A>G',\n",
       "      'hgvs_p': 'p.Lys218Glu',\n",
       "      'protein': {'length': '238', 'position': '218'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '5',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '970'},\n",
       "      'cds': {'length': '795', 'position': '730'},\n",
       "      'effect': 'missense_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.730A>G',\n",
       "      'hgvs_p': 'p.Lys244Glu',\n",
       "      'protein': {'length': '264', 'position': '244'},\n",
       "      'putative_impact': 'MODERATE',\n",
       "      'rank': '6',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2353',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4848',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2451',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2451',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'G', 'position': '56365344', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56363249C>T',\n",
       "   '_score': 2.3442407,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'C',\n",
       "    'annotype': ['Intergenic', 'Transcript'],\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.063,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.268,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.583,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['upstream', 'intron'],\n",
       "    'consequence': ['UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': 1.9, 'mgw': 0.21, 'prot': -3.69, 'roll': 0.38},\n",
       "    'encode': {'exp': 95.21,\n",
       "     'h3k27ac': 4.68,\n",
       "     'h3k4me1': 8.64,\n",
       "     'h3k4me3': 4.4,\n",
       "     'nucleo': 1.7},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'fitcons': 0.117096,\n",
       "    'gc': 0.53,\n",
       "    'gene': [{'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'}],\n",
       "    'gerp': {'n': 4.78, 's': 2.43},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 419,\n",
       "    'min_dist_tss': 2003,\n",
       "    'mutindex': 76,\n",
       "    'phast_cons': {'mammalian': 0.016, 'primate': 0.008, 'vertebrate': 0.688},\n",
       "    'phred': 5.714,\n",
       "    'phylop': {'mammalian': 0.553, 'primate': -1.142, 'vertebrate': 0.786},\n",
       "    'pos': 56363249,\n",
       "    'rawscore': 0.302061,\n",
       "    'ref': 'C',\n",
       "    'segway': 'TF2',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'C', 'freq': 0.9996},\n",
       "     {'allele': 'T', 'freq': 0.0003994}],\n",
       "    'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 138,\n",
       "    'flags': ['ASP', 'INT', 'KGPhase3'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'gmaf': 0.0003994,\n",
       "    'hg19': {'end': 56363249, 'start': 56363249},\n",
       "    'ref': 'C',\n",
       "    'rsid': 'rs377191803',\n",
       "    'validated': True,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'C': 4406, 'T': 0},\n",
       "     'all': {'C': 13005, 'T': 1},\n",
       "     'european_american': {'C': 8599, 'T': 1}},\n",
       "    'alt': 'T',\n",
       "    'avg_sample_read': 65,\n",
       "    'chimp_allele': 'C',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 2.4, 'phast_cons': 0.0},\n",
       "    'dbsnp_version': 138,\n",
       "    'estimated_age_kyrs': {'ea': '1.2+/-3.3'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'CC': 2203, 'TC': 0, 'TT': 0},\n",
       "     'all_genotype': {'CC': 6502, 'TC': 1, 'TT': 0},\n",
       "     'european_american': {'CC': 4299, 'TC': 1, 'TT': 0}},\n",
       "    'hg19': {'end': 56363249, 'start': 56363249},\n",
       "    'hg38': {'end': 55969465, 'start': 55969465},\n",
       "    'hgvs': [{'coding': 'c.486+517C>T'}, {'coding': 'c.487-10C>T'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0,\n",
       "     'all': 0.0077,\n",
       "     'european_american': 0.0116},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'C',\n",
       "    'ref_base_ncbi': 'C',\n",
       "    'rsid': 'rs377191803'},\n",
       "   'exac': {'ac': {'ac': 6,\n",
       "     'ac_adj': 6,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 1,\n",
       "     'ac_female': 2,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 6,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 4,\n",
       "     'ac_nfe': 5,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 4.942e-05,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 117554,\n",
       "     'an_afr': 10344,\n",
       "     'an_amr': 11506,\n",
       "     'an_eas': 8648,\n",
       "     'an_female': 52974,\n",
       "     'an_fin': 6574,\n",
       "     'an_male': 64580,\n",
       "     'an_nfe': 65646,\n",
       "     'an_oth': 874,\n",
       "     'an_sas': 13962},\n",
       "    'baseqranksum': 0.517,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.337,\n",
       "    'culprit': 'FS',\n",
       "    'dp': 1654642,\n",
       "    'fs': 0.0,\n",
       "    'het': {'ac_het': 6,\n",
       "     'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 1,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 5,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 0,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0005,\n",
       "    'mq': {'mq': 59.6, 'mq0': 0, 'mqranksum': 0.257},\n",
       "    'ncc': 3,\n",
       "    'pos': 56363249,\n",
       "    'qd': 12.61,\n",
       "    'qual': 7239.08,\n",
       "    'readposranksum': 0.337,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.53},\n",
       "   'exac_nontcga': {'ac': {'ac': 6,\n",
       "     'ac_adj': 6,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 1,\n",
       "     'ac_female': 2,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 6,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 4,\n",
       "     'ac_nfe': 5,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 5.649e-05,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 102390,\n",
       "     'an_afr': 9004,\n",
       "     'an_amr': 11146,\n",
       "     'an_eas': 7864,\n",
       "     'an_female': 44732,\n",
       "     'an_fin': 6574,\n",
       "     'an_male': 57658,\n",
       "     'an_nfe': 53284,\n",
       "     'an_oth': 660,\n",
       "     'an_sas': 13858},\n",
       "    'baseqranksum': 0.517,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.337,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 0.0,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 1,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 5,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0005,\n",
       "    'mq': {'mq': 59.6, 'mq0': 0, 'mqranksum': 0.257},\n",
       "    'ncc': 3,\n",
       "    'pos': 56363249,\n",
       "    'qd': 12.61,\n",
       "    'readposranksum': 0.337,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.53},\n",
       "   'hg19': {'end': 56363249, 'start': 56363249},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '3403',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4448',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3403',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2753',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4546',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4546',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.487-10C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+517C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+517C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'T', 'position': '56363249', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56362536A>G',\n",
       "   '_score': 2.339395,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'A',\n",
       "    'annotype': ['Intergenic', 'Transcript'],\n",
       "    'bstatistic': 252,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.071,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.016,\n",
       "     'tssaflnk': 0.008,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.15,\n",
       "     'txflnk': 0.039,\n",
       "     'txwk': 0.646,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['upstream', 'intron'],\n",
       "    'consequence': ['UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [1, 2],\n",
       "    'cpg': 0.01,\n",
       "    'dna': {'helt': -0.32, 'mgw': 0.34, 'prot': 4.56, 'roll': 1.77},\n",
       "    'encode': {'exp': 206.57,\n",
       "     'h3k27ac': 8.8,\n",
       "     'h3k4me1': 16.24,\n",
       "     'h3k4me3': 17.6,\n",
       "     'nucleo': 1.1},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'fitcons': 0.117096,\n",
       "    'gc': 0.47,\n",
       "    'gene': [{'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'}],\n",
       "    'gerp': {'n': 3, 's': 0.471},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 441,\n",
       "    'min_dist_tss': 1290,\n",
       "    'mutindex': -69,\n",
       "    'phast_cons': {'mammalian': 0.002, 'primate': 0.198, 'vertebrate': 0.001},\n",
       "    'phred': 7.617,\n",
       "    'phylop': {'mammalian': -0.004, 'primate': 0.375, 'vertebrate': 0.224},\n",
       "    'pos': 56362536,\n",
       "    'rawscore': 0.524706,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'A'}, {'allele': 'G'}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 138,\n",
       "    'flags': ['ASP', 'INT'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56362536, 'start': 56362536},\n",
       "    'ref': 'A',\n",
       "    'rsid': 'rs376207715',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'A': 4404, 'G': 2},\n",
       "     'all': {'A': 13004, 'G': 2},\n",
       "     'european_american': {'A': 8600, 'G': 0}},\n",
       "    'alt': 'G',\n",
       "    'avg_sample_read': 83,\n",
       "    'chimp_allele': 'A',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 0.5, 'phast_cons': 0.0},\n",
       "    'dbsnp_version': 138,\n",
       "    'estimated_age_kyrs': {'aa': '4.2+/-10.8'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'AA': 2201, 'GA': 2, 'GG': 0},\n",
       "     'all_genotype': {'AA': 6501, 'GA': 2, 'GG': 0},\n",
       "     'european_american': {'AA': 4300, 'GA': 0, 'GG': 0}},\n",
       "    'hg19': {'end': 56362536, 'start': 56362536},\n",
       "    'hg38': {'end': 55968752, 'start': 55968752},\n",
       "    'hgvs': {'coding': 'c.316-26A>G'},\n",
       "    'ma_fin_percent': {'african_american': 0.0454,\n",
       "     'all': 0.0154,\n",
       "     'european_american': 0.0},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'A',\n",
       "    'ref_base_ncbi': 'A',\n",
       "    'rsid': 'rs376207715'},\n",
       "   'hg19': {'end': 56362536, 'start': 56362536},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '2690',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2690',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2040',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.316-26A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.238-26A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '2',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.316-26A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'G', 'position': '56362536', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56365319C>T',\n",
       "   '_score': 2.3220916,\n",
       "   'cadd': {'1000g': {'af': 0.0},\n",
       "    '_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'C',\n",
       "    'annotype': ['CodingTranscript', 'Intergenic', 'Intergenic', 'Transcript'],\n",
       "    'bstatistic': 238,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.165,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.244,\n",
       "     'txflnk': 0.047,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['synonymous', 'upstream', 'upstream', 'intron'],\n",
       "    'consequence': ['SYNONYMOUS', 'UPSTREAM', 'UPSTREAM', 'INTRONIC'],\n",
       "    'consscore': [5, 1, 1, 2],\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': 1.51, 'mgw': 0.39, 'prot': -2.95, 'roll': 0.72},\n",
       "    'dst2splice': 15,\n",
       "    'dst2spltype': 'ACCEPTOR',\n",
       "    'encode': {'exp': 313.78,\n",
       "     'h3k27ac': 26.6,\n",
       "     'h3k4me1': 17.08,\n",
       "     'h3k4me3': 14.84,\n",
       "     'nucleo': 1.3},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.001},\n",
       "    'exon': '7/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.52,\n",
       "    'gene': [{'ccds_id': 'CCDS8898.1',\n",
       "      'cds': {'cdna_pos': 1047,\n",
       "       'cds_pos': 807,\n",
       "       'rel_cdna_pos': 0.46,\n",
       "       'rel_cds_pos': 0.9},\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2',\n",
       "      'prot': {'domain': 'ndomain', 'protpos': 269, 'rel_prot_pos': 0.9}},\n",
       "     {'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'feature_id': 'ENST00000549233',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'}],\n",
       "    'gerp': {'n': 5.37, 'rs': 466.9, 'rs_pval': 8.6458e-62, 's': -3.46},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 91,\n",
       "    'min_dist_tss': 415,\n",
       "    'mutindex': 105,\n",
       "    'naa': 'Y',\n",
       "    'oaa': 'Y',\n",
       "    'phast_cons': {'mammalian': 0.974, 'primate': 0.086, 'vertebrate': 0.956},\n",
       "    'phred': 6.633,\n",
       "    'phylop': {'mammalian': -0.641, 'primate': -0.961, 'vertebrate': -0.323},\n",
       "    'pos': 56365319,\n",
       "    'rawscore': 0.402155,\n",
       "    'ref': 'C',\n",
       "    'segway': 'GM1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'C', 'freq': 0.9998},\n",
       "     {'allele': 'T', 'freq': 0.0001997}],\n",
       "    'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 134,\n",
       "    'flags': ['ASP', 'KGPhase1', 'KGPhase3', 'REF', 'S3D', 'SYN'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'gmaf': 0.0001997,\n",
       "    'hg19': {'end': 56365319, 'start': 56365319},\n",
       "    'ref': 'C',\n",
       "    'rsid': 'rs144972287',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'C': 4406, 'T': 0},\n",
       "     'all': {'C': 13000, 'T': 6},\n",
       "     'european_american': {'C': 8594, 'T': 6}},\n",
       "    'alt': 'T',\n",
       "    'avg_sample_read': 127,\n",
       "    'chimp_allele': 'C',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': -3.5, 'phast_cons': 1.0},\n",
       "    'dbsnp_version': 134,\n",
       "    'estimated_age_kyrs': {'ea': '5.0+/-18.6'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'coding-synonymous',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'CC': 2203, 'TC': 0, 'TT': 0},\n",
       "     'all_genotype': {'CC': 6497, 'TC': 6, 'TT': 0},\n",
       "     'european_american': {'CC': 4294, 'TC': 6, 'TT': 0}},\n",
       "    'hg19': {'end': 56365319, 'start': 56365319},\n",
       "    'hg38': {'end': 55971535, 'start': 55971535},\n",
       "    'hgvs': [{'coding': 'c.705C>T', 'protein': 'p.(Y235=)'},\n",
       "     {'coding': 'c.807C>T', 'protein': 'p.(Y269=)'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0,\n",
       "     'all': 0.0461,\n",
       "     'european_american': 0.0698},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'C',\n",
       "    'ref_base_ncbi': 'C',\n",
       "    'rsid': 'rs144972287'},\n",
       "   'exac': {'ac': {'ac': 27,\n",
       "     'ac_adj': 27,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 11,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 27,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 16,\n",
       "     'ac_nfe': 27,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.0002224,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121372,\n",
       "     'an_afr': 10406,\n",
       "     'an_amr': 11578,\n",
       "     'an_eas': 8654,\n",
       "     'an_female': 54100,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 67272,\n",
       "     'an_nfe': 66702,\n",
       "     'an_oth': 908,\n",
       "     'an_sas': 16510},\n",
       "    'baseqranksum': -2.253,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.349,\n",
       "    'culprit': 'FS',\n",
       "    'dp': 2331217,\n",
       "    'fs': 0.0,\n",
       "    'het': {'ac_het': 27,\n",
       "     'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 27,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 0,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0003,\n",
       "    'mq': {'mq': 59.5, 'mq0': 0, 'mqranksum': 0.189},\n",
       "    'ncc': 0,\n",
       "    'pos': 56365319,\n",
       "    'qd': 12.64,\n",
       "    'qual': 75409.64,\n",
       "    'readposranksum': 0.531,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.34},\n",
       "   'exac_nontcga': {'ac': {'ac': 23,\n",
       "     'ac_adj': 23,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 9,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 23,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 14,\n",
       "     'ac_nfe': 23,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.0002166,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 106172,\n",
       "     'an_afr': 9066,\n",
       "     'an_amr': 11216,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45852,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 60320,\n",
       "     'an_nfe': 54310,\n",
       "     'an_oth': 694,\n",
       "     'an_sas': 16406},\n",
       "    'baseqranksum': -2.253,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.349,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 0.0,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 23,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': -0.0003,\n",
       "    'mq': {'mq': 59.5, 'mq0': 0, 'mqranksum': 0.189},\n",
       "    'ncc': 0,\n",
       "    'pos': 56365319,\n",
       "    'qd': 12.64,\n",
       "    'readposranksum': 0.531,\n",
       "    'ref': 'C',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.34},\n",
       "   'geno2mp': {'hpo_count': 3},\n",
       "   'hg19': {'end': 56365319, 'start': 56365319},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '1047'},\n",
       "      'cds': {'length': '897', 'position': '807'},\n",
       "      'effect': 'synonymous_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.807C>T',\n",
       "      'hgvs_p': 'p.Tyr269Tyr',\n",
       "      'protein': {'length': '298', 'position': '269'},\n",
       "      'putative_impact': 'LOW',\n",
       "      'rank': '7',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '867'},\n",
       "      'cds': {'length': '717', 'position': '627'},\n",
       "      'effect': 'synonymous_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.627C>T',\n",
       "      'hgvs_p': 'p.Tyr209Tyr',\n",
       "      'protein': {'length': '238', 'position': '209'},\n",
       "      'putative_impact': 'LOW',\n",
       "      'rank': '5',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '945'},\n",
       "      'cds': {'length': '795', 'position': '705'},\n",
       "      'effect': 'synonymous_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.705C>T',\n",
       "      'hgvs_p': 'p.Tyr235Tyr',\n",
       "      'protein': {'length': '264', 'position': '235'},\n",
       "      'putative_impact': 'LOW',\n",
       "      'rank': '6',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2378',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4823',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2476',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2476',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'T', 'position': '56365319', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56365280A>G',\n",
       "   '_score': 2.306868,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'A',\n",
       "    'annotype': ['Intergenic', 'Transcript', 'Transcript'],\n",
       "    'bstatistic': 238,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.165,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.244,\n",
       "     'txflnk': 0.047,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['upstream', 'intron', 'intron'],\n",
       "    'consequence': ['UPSTREAM', 'INTRONIC', 'INTRONIC'],\n",
       "    'consscore': [1, 2, 2],\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': 0.6, 'mgw': 0.42, 'prot': 2.08, 'roll': 1.67},\n",
       "    'encode': {'exp': 337.38,\n",
       "     'h3k27ac': 27.48,\n",
       "     'h3k4me1': 12.68,\n",
       "     'h3k4me3': 12.16,\n",
       "     'nucleo': 1.7},\n",
       "    'esp': {'af': 0.0, 'afr': 0.0, 'eur': 0.0},\n",
       "    'fitcons': 0.156329,\n",
       "    'gc': 0.5,\n",
       "    'gene': [{'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'},\n",
       "     {'feature_id': 'ENST00000549233',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'}],\n",
       "    'gerp': {'n': 5.15, 's': 1.02},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 130,\n",
       "    'min_dist_tss': 376,\n",
       "    'mutindex': -42,\n",
       "    'phast_cons': {'mammalian': 0.705, 'primate': 0.247, 'vertebrate': 0.469},\n",
       "    'phred': 12.47,\n",
       "    'phylop': {'mammalian': 0.332, 'primate': 0.478, 'vertebrate': 0.29},\n",
       "    'pos': 56365280,\n",
       "    'rawscore': 1.33878,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GM0',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'A'}, {'allele': 'G'}],\n",
       "    'alt': 'G',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 138,\n",
       "    'flags': ['ASP', 'INT'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'hg19': {'end': 56365280, 'start': 56365280},\n",
       "    'ref': 'A',\n",
       "    'rsid': 'rs369152141',\n",
       "    'validated': True,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'A': 4406, 'G': 0},\n",
       "     'all': {'A': 13005, 'G': 1},\n",
       "     'european_american': {'A': 8599, 'G': 1}},\n",
       "    'alt': 'G',\n",
       "    'avg_sample_read': 87,\n",
       "    'chimp_allele': 'A',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 1.0, 'phast_cons': 0.7},\n",
       "    'dbsnp_version': 138,\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'AA': 2203, 'GA': 0, 'GG': 0},\n",
       "     'all_genotype': {'AA': 6502, 'GA': 1, 'GG': 0},\n",
       "     'european_american': {'AA': 4299, 'GA': 1, 'GG': 0}},\n",
       "    'hg19': {'end': 56365280, 'start': 56365280},\n",
       "    'hg38': {'end': 55971496, 'start': 55971496},\n",
       "    'hgvs': [{'coding': 'c.691-25A>G'}, {'coding': 'c.793-25A>G'}],\n",
       "    'ma_fin_percent': {'african_american': 0.0,\n",
       "     'all': 0.0077,\n",
       "     'european_american': 0.0116},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'A',\n",
       "    'ref_base_ncbi': 'A',\n",
       "    'rsid': 'rs369152141'},\n",
       "   'exac': {'ac': {'ac': 22,\n",
       "     'ac_adj': 22,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 6,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 22,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 16,\n",
       "     'ac_nfe': 22,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.0001812,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121114,\n",
       "     'an_afr': 10400,\n",
       "     'an_amr': 11558,\n",
       "     'an_eas': 8652,\n",
       "     'an_female': 54000,\n",
       "     'an_fin': 6612,\n",
       "     'an_male': 67114,\n",
       "     'an_nfe': 66592,\n",
       "     'an_oth': 906,\n",
       "     'an_sas': 16394},\n",
       "    'baseqranksum': 4.18,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.259,\n",
       "    'culprit': 'FS',\n",
       "    'dp': 2318515,\n",
       "    'fs': 0.0,\n",
       "    'het': {'ac_het': 22,\n",
       "     'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 22,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 0,\n",
       "     'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0885,\n",
       "    'mq': {'mq': 59.67, 'mq0': 0, 'mqranksum': -0.043},\n",
       "    'ncc': 3,\n",
       "    'pos': 56365280,\n",
       "    'qd': 17.34,\n",
       "    'qual': 43660.48,\n",
       "    'readposranksum': 0.326,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.77},\n",
       "   'exac_nontcga': {'ac': {'ac': 20,\n",
       "     'ac_adj': 20,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 5,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 20,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 15,\n",
       "     'ac_nfe': 20,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.0001883,\n",
       "    'alleles': 'G',\n",
       "    'alt': 'G',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 105922,\n",
       "     'an_afr': 9062,\n",
       "     'an_amr': 11196,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45756,\n",
       "     'an_fin': 6612,\n",
       "     'an_male': 60166,\n",
       "     'an_nfe': 54204,\n",
       "     'an_oth': 692,\n",
       "     'an_sas': 16290},\n",
       "    'baseqranksum': 4.18,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.259,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 0.0,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 20,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0885,\n",
       "    'mq': {'mq': 59.67, 'mq0': 0, 'mqranksum': -0.043},\n",
       "    'ncc': 3,\n",
       "    'pos': 56365280,\n",
       "    'qd': 17.34,\n",
       "    'readposranksum': 0.326,\n",
       "    'ref': 'A',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 5.77},\n",
       "   'geno2mp': {'hpo_count': 2},\n",
       "   'hg19': {'end': 56365280, 'start': 56365280},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '2417',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4784',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2515',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2515',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.793-25A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '6',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.613-25A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.691-25A>G',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '5',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'G', 'position': '56365280', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56364793G>A',\n",
       "   '_score': 2.28911,\n",
       "   'cadd': {'1000g': {'af': 0.01, 'afr': 0.03, 'amr': 0.003, 'eur': 0.001},\n",
       "    '_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'A',\n",
       "    'anc': 'G',\n",
       "    'annotype': ['RegulatoryFeature',\n",
       "     'Intergenic',\n",
       "     'Intergenic',\n",
       "     'Intergenic',\n",
       "     'Transcript'],\n",
       "    'bstatistic': 242,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.094,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.307,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.512,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['regulatory', 'upstream', 'upstream', 'upstream', 'intron'],\n",
       "    'consequence': ['REGULATORY',\n",
       "     'UPSTREAM',\n",
       "     'UPSTREAM',\n",
       "     'UPSTREAM',\n",
       "     'INTRONIC'],\n",
       "    'consscore': [4, 1, 1, 1, 2],\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': -1.98, 'mgw': -0.28, 'prot': 2.16, 'roll': -1.33},\n",
       "    'encode': {'exp': 112.6,\n",
       "     'h3k27ac': 7.6,\n",
       "     'h3k4me1': 13.88,\n",
       "     'h3k4me3': 6.8,\n",
       "     'nucleo': 0.7},\n",
       "    'esp': {'af': 0.011, 'afr': 0.032, 'eur': 0.0},\n",
       "    'fitcons': 0.54483,\n",
       "    'gc': 0.5,\n",
       "    'gene': [{'feature_id': 'ENSR00000653685'},\n",
       "     {'ccds_id': 'CCDS8900.1',\n",
       "      'feature_id': 'ENST00000360299',\n",
       "      'gene_id': 'ENSG00000111540',\n",
       "      'genename': 'RAB5B'},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'},\n",
       "     {'ccds_id': 'CCDS8898.1',\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2'}],\n",
       "    'gerp': {'n': 4.7, 'rs': 2645.2, 'rs_pval': 9.82059e-310, 's': 0.898},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'TRUE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 79,\n",
       "    'min_dist_tss': 112,\n",
       "    'mutindex': 79,\n",
       "    'phast_cons': {'mammalian': 0.027, 'primate': 0.102, 'vertebrate': 0.001},\n",
       "    'phred': 0.109,\n",
       "    'phylop': {'mammalian': 0.329, 'primate': -0.917, 'vertebrate': 0.188},\n",
       "    'pos': 56364793,\n",
       "    'rawscore': -0.624039,\n",
       "    'ref': 'G',\n",
       "    'segway': 'TF1',\n",
       "    'tf': {'bs': 3, 'bs_peaks': 3, 'bs_peaks_max': 223.6802},\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'1000gp3': {'ac': 57,\n",
       "     'af': 0.011381789137380192,\n",
       "     'afr_ac': 54,\n",
       "     'afr_af': 0.04084720121028744,\n",
       "     'amr_ac': 2,\n",
       "     'amr_af': 0.002881844380403458,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0.0,\n",
       "     'eur_ac': 1,\n",
       "     'eur_af': 0.0009940357852882703,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0.0},\n",
       "    'aa': {'alt': 'M',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 2,\n",
       "     'pos': 10,\n",
       "     'ref': 'T',\n",
       "     'refcodon': 'ACG'},\n",
       "    'alt': 'A',\n",
       "    'ancestral_allele': 'G',\n",
       "    'cds_strand': '-',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.1693,\n",
       "     'score': 0.8779750316629121},\n",
       "    'eigen': {'phred': 0.6746487,\n",
       "     'raw': -0.786467449293428,\n",
       "     'raw_rankscore': 0.137},\n",
       "    'eigen-pc': {'raw': -0.920892943805372, 'raw_rankscore': 0.11403},\n",
       "    'ensembl': {'geneid': 'ENSG00000185664',\n",
       "     'proteinid': 'ENSP00000448871',\n",
       "     'transcriptid': 'ENST00000549233'},\n",
       "    'esp6500': {'aa_ac': 142,\n",
       "     'aa_af': 0.03222877893781208,\n",
       "     'ea_ac': 2,\n",
       "     'ea_af': 0.00023255813953488373},\n",
       "    'exac': {'ac': 382,\n",
       "     'adj_ac': 382,\n",
       "     'adj_af': 0.003153,\n",
       "     'af': 0.003146,\n",
       "     'afr_ac': 368,\n",
       "     'afr_af': 0.03595,\n",
       "     'amr_ac': 6,\n",
       "     'amr_af': 0.0005183,\n",
       "     'eas_ac': 2,\n",
       "     'eas_af': 0.0002312,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 6,\n",
       "     'nfe_af': 9e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'exac_nonpsych': {'ac': 381,\n",
       "     'adj_ac': 381,\n",
       "     'adj_af': 0.00421,\n",
       "     'af': 0.004198,\n",
       "     'afr_ac': 367,\n",
       "     'afr_af': 0.03592,\n",
       "     'amr_ac': 6,\n",
       "     'amr_af': 0.0005189,\n",
       "     'eas_ac': 2,\n",
       "     'eas_af': 0.0003549,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 6,\n",
       "     'nfe_af': 0.0001426},\n",
       "    'exac_nontcga': {'ac': 329,\n",
       "     'adj_ac': 329,\n",
       "     'adj_af': 0.003104,\n",
       "     'af': 0.003098,\n",
       "     'afr_ac': 321,\n",
       "     'afr_af': 0.03608,\n",
       "     'amr_ac': 5,\n",
       "     'amr_af': 0.0004458,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 3,\n",
       "     'nfe_af': 5.526e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFDBHCIJ',\n",
       "     'coding_pred': 'N',\n",
       "     'coding_rankscore': 0.07892,\n",
       "     'coding_score': 0.03074},\n",
       "    'genename': 'PMEL',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.99999999412763},\n",
       "    'gerp++': {'nr': 4.7, 'rs': 0.898, 'rs_rankscore': 0.18319},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.99779,\n",
       "     'fitcons_score': 0.819298},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.06112,\n",
       "     'fitcons_score': 0.463624},\n",
       "    'hg18': {'end': 54651060, 'start': 54651060},\n",
       "    'hg19': {'end': 56364793, 'start': 56364793},\n",
       "    'hg38': {'end': 55971009, 'start': 55971009},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.06948,\n",
       "     'fitcons_score': 0.491896},\n",
       "    'integrated': {'confidence_value': 2,\n",
       "     'fitcons_rankscore': 0.22523,\n",
       "     'fitcons_score': 0.549461},\n",
       "    'metalr': {'pred': 'T', 'rankscore': 0.17827, 'score': 0.0414},\n",
       "    'metasvm': {'pred': 'T', 'rankscore': 0.21055, 'score': -1.0281},\n",
       "    'mutationtaster': {'AAE': ['.', '.', '.', '.'],\n",
       "     'converted_rankscore': 0.08979,\n",
       "     'model': ['without_aae', 'without_aae', 'without_aae', 'without_aae'],\n",
       "     'pred': ['N', 'N', 'N', 'N'],\n",
       "     'score': [0.999999, 0.999999, 0.999999, 0.999999]},\n",
       "    'phastcons': {'100way': {'vertebrate': 0.0,\n",
       "      'vertebrate_rankscore': 0.06329},\n",
       "     '20way': {'mammalian': 0.943, 'mammalian_rankscore': 0.41702}},\n",
       "    'phylo': {'p100way': {'vertebrate': 0.02, 'vertebrate_rankscore': 0.13254},\n",
       "     'p20way': {'mammalian': -0.306, 'mammalian_rankscore': 0.0795}},\n",
       "    'provean': {'pred': 'N', 'rankscore': 0.032, 'score': 0.45},\n",
       "    'ref': 'G',\n",
       "    'reliability_index': 5,\n",
       "    'rsid': 'rs114405105',\n",
       "    'sift': {'converted_rankscore': 0.49066, 'pred': 'D', 'score': 0.021},\n",
       "    'siphy_29way': {'logodds': 6.7789,\n",
       "     'logodds_rankscore': 0.22671,\n",
       "     'pi': {'a': 0.3641, 'c': 0.0, 'g': 0.6359, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F',\n",
       "     'rankscore': 0.109,\n",
       "     'score': 0.109,\n",
       "     'transcriptid': 'ENST00000549233',\n",
       "     'transcriptvar': 'T10M'}},\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'G', 'freq': 0.9886},\n",
       "     {'allele': 'A', 'freq': 0.01138}],\n",
       "    'alt': 'A',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 132,\n",
       "    'flags': ['ASP', 'HD', 'INT', 'KGPhase1', 'KGPhase3'],\n",
       "    'gene': {'geneid': '1017', 'symbol': 'CDK2'},\n",
       "    'gmaf': 0.01138,\n",
       "    'hg19': {'end': 56364793, 'start': 56364793},\n",
       "    'ref': 'G',\n",
       "    'rsid': 'rs114405105',\n",
       "    'validated': True,\n",
       "    'var_subtype': 'ts',\n",
       "    'vartype': 'snp'},\n",
       "   'evs': {'allele_count': {'african_american': {'A': 142, 'G': 4264},\n",
       "     'all': {'A': 144, 'G': 12862},\n",
       "     'european_american': {'A': 2, 'G': 8598}},\n",
       "    'alt': 'A',\n",
       "    'avg_sample_read': 69,\n",
       "    'chimp_allele': 'G',\n",
       "    'chrom': 12,\n",
       "    'coding_dna_size': [795, 897],\n",
       "    'conservation': {'gerp': 0.9, 'phast_cons': 0.0},\n",
       "    'dbsnp_version': 132,\n",
       "    'estimated_age_kyrs': {'aa': '99.4+/-138.0', 'ea': '1.2+/-3.3'},\n",
       "    'filter_status': 'PASS',\n",
       "    'function_gvs': 'intron',\n",
       "    'gene': [{'accession': 'NM_052827.2', 'symbol': 'CDK2'},\n",
       "     {'accession': 'NM_001798.3', 'symbol': 'CDK2'}],\n",
       "    'genotype_count': {'african_american': {'AA': 1, 'AG': 140, 'GG': 2062},\n",
       "     'all_genotype': {'AA': 1, 'AG': 142, 'GG': 6360},\n",
       "     'european_american': {'AA': 0, 'AG': 2, 'GG': 4298}},\n",
       "    'hg19': {'end': 56364793, 'start': 56364793},\n",
       "    'hg38': {'end': 55971009, 'start': 55971009},\n",
       "    'hgvs': [{'coding': 'c.487-35G>A'}, {'coding': 'c.589-35G>A'}],\n",
       "    'ma_fin_percent': {'african_american': 3.2229,\n",
       "     'all': 1.1072,\n",
       "     'european_american': 0.0233},\n",
       "    'on_illumina_human_exome_chip': 'no',\n",
       "    'polyphen2': {'class': 'u', 'score': 'n'},\n",
       "    'ref': 'G',\n",
       "    'ref_base_ncbi': 'G',\n",
       "    'rsid': 'rs114405105'},\n",
       "   'exac': {'ac': {'ac': 382,\n",
       "     'ac_adj': 382,\n",
       "     'ac_afr': 368,\n",
       "     'ac_amr': 6,\n",
       "     'ac_eas': 2,\n",
       "     'ac_female': 238,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 376,\n",
       "     'ac_hom': 3,\n",
       "     'ac_male': 144,\n",
       "     'ac_nfe': 6,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.003146,\n",
       "    'alleles': 'A',\n",
       "    'alt': 'A',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121168,\n",
       "     'an_afr': 10236,\n",
       "     'an_amr': 11576,\n",
       "     'an_eas': 8652,\n",
       "     'an_female': 53926,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 67242,\n",
       "     'an_nfe': 66670,\n",
       "     'an_oth': 908,\n",
       "     'an_sas': 16512},\n",
       "    'baseqranksum': 4.5,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.3,\n",
       "    'culprit': 'FS',\n",
       "    'dp': 1535716,\n",
       "    'fs': 1.772,\n",
       "    'het': {'ac_het': 376,\n",
       "     'het_afr': 362,\n",
       "     'het_amr': 6,\n",
       "     'het_eas': 2,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 6,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'ac_hom': 3,\n",
       "     'hom_afr': 3,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0127,\n",
       "    'mq': {'mq': 59.79, 'mq0': 0, 'mqranksum': 0.245},\n",
       "    'ncc': 3,\n",
       "    'pos': 56364793,\n",
       "    'qd': 14.07,\n",
       "    'qual': 753338.73,\n",
       "    'readposranksum': 0.658,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 4.22},\n",
       "   'exac_nontcga': {'ac': {'ac': 329,\n",
       "     'ac_adj': 329,\n",
       "     'ac_afr': 321,\n",
       "     'ac_amr': 5,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 206,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 323,\n",
       "     'ac_hom': 3,\n",
       "     'ac_male': 123,\n",
       "     'ac_nfe': 3,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 0.003098,\n",
       "    'alleles': 'A',\n",
       "    'alt': 'A',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 105984,\n",
       "     'an_afr': 8896,\n",
       "     'an_amr': 11216,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45692,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 60292,\n",
       "     'an_nfe': 54290,\n",
       "     'an_oth': 694,\n",
       "     'an_sas': 16408},\n",
       "    'baseqranksum': 4.5,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.3,\n",
       "    'culprit': 'FS',\n",
       "    'fs': 1.772,\n",
       "    'het': {'het_afr': 315,\n",
       "     'het_amr': 5,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 3,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 3,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0127,\n",
       "    'mq': {'mq': 59.79, 'mq0': 0, 'mqranksum': 0.245},\n",
       "    'ncc': 3,\n",
       "    'pos': 56364793,\n",
       "    'qd': 14.07,\n",
       "    'readposranksum': 0.658,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 4.22},\n",
       "   'geno2mp': {'hpo_count': 2},\n",
       "   'hg19': {'end': 56364793, 'start': 56364793},\n",
       "   'snpeff': {'ann': [{'distance_to_feature': '4947',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2904',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4947',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4297',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3002',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3002',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.589-35G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '5',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.409-35G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.487-35G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}]},\n",
       "   'vcf': {'alt': 'A', 'position': '56364793', 'ref': 'G'}}],\n",
       " 'max_score': 2.3620462,\n",
       " 'took': 4,\n",
       " 'total': 591}"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query_by_uri('http://localhost:8899/symbol/', 'cdk2',context)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
